### PR TITLE
feat: design fixes

### PR DESCRIPTION
**Pull request by tlebon**
includes
- [x]  a handful of design fixes
- [x] content fixes
- [x] bug fixes
- [x] rounding issue on portfolio
- [x] bet history column change

please refer to this task to see the individual changes
https://app.asana.com/0/1199218544636757/1200830815509711/f

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -173,5 +173,7 @@
   "signOut": "Sign Out",
   "availableBalance": "Available Balance",
   "openPositions": "Open Positions",
-  "reset": "Reset"
+  "reset": "Reset",
+  "connectWalletBuy": "Connect Wallet + Buy",
```
Comment (shekhar-shubhendu) [inline public/locales/en/common.json]: we can not do `+` actions with torus
so if the wallet is not connected it should be `Connect wallet to continue`

File: src/design-system/molecules/MarketDetailCard/MarketDetailCard.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export const MarketDetailCard: React.FC<MarketDetailCardProps> = ({ title, items
       >
         {titleHeader}
       </AccordionSummary>
-      <AccordionDetails>{bodyContent}</AccordionDetails>
+      <AccordionDetails sx={{ paddingTop: '0.25rem' }}>{bodyContent}</AccordionDetails>
```
Comment (shekhar-shubhendu) [inline src/design-system/molecules/MarketDetailCard/MarketDetailCard.tsx]: use this from a constant

File: public/locales/en/common.json (hunk 1 above)
Comment (tlebon) [reply public/locales/en/common.json]: can we do this with another if clause instead `(if torus){...do things}`? i would put three options instead- as the current design shows this updated text.

@shekhar-shubhendu

Comment (tlebon) [reply public/locales/en/common.json]: i also dont think the + text necessarily implies that the action will occur immediately but maybe @anitasengupta would feel differently

File: src/design-system/molecules/MarketDetailCard/MarketDetailCard.tsx (hunk 2 above)
Comment (tlebon) [reply src/design-system/molecules/MarketDetailCard/MarketDetailCard.tsx]: i was thinking sx behaved more like mui props so it wouldn't rerender but this is fine I've moved this now.

File: public/locales/en/common.json (hunk 1 above)
Comment (shekhar-shubhendu) [reply public/locales/en/common.json]: I do not want to have if(torus) before every action as we might need to integrate with other providers in the future.
wallet integration should not be tightly coupled.

Comment (tlebon) [reply public/locales/en/common.json]: this is changed

**Review by shekhar-shubhendu — APPROVED** [review]

**Pull request merged**